### PR TITLE
Fix tests compatibility with PasteDeploy 2+

### PR DESCRIPTION
**Pull request by onovy**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def readfile(name):
 changes = readfile('CHANGES.rst')
 
 install_requires = [
-    'PasteDeploy >= 1.5.0',  # py3 compat
+    'PasteDeploy >= 2.0',  # py3 compat
```
Comment (mmerickel) [inline setup.py]: the pin should be adjusted to indicate why it's pinned to 2.0. Likely something along the lines of "default override order".

**Comment by onovy** [comment]
done

**Comment by mmerickel** [comment]
Thanks @onovy!

**Comment by Ketchup901** [comment]
Is this going to be merged?

**Pull request merged**